host-configs/LLNL/quartz-base.cmake

```diff
@@ -31,3 +31,4 @@ set(ENABLE_PAMELA ON CACHE BOOL "")
 set(ENABLE_PVTPackage ON CACHE BOOL "")
 set(ENABLE_GEOSX_PTP ON CACHE BOOL "" FORCE)
 set(ENABLE_PETSC OFF CACHE BOOL "Enables PETSc." FORCE)
+set(ENABLE_DOXYGEN OFF CACHE BOOL "" FORCE)
```

Comment:
What's the motivation for disabling doxygen on quartz?

Reply:
Somehow the path to doxygen is wrong so it cannot find the executable. I tried to understand why but I wasn't able to so I just disabled it. @rrsettgast has fixed this already in another PR. It will come back.